specs/RuntimeSpec/Fees/Fees.md

```diff
@@ -73,7 +68,40 @@ Here is the list of actions and their corresponding fees:
     - the base fee [`delete_key_cost`](/GenesisConfig/RuntimeFeeConfig/ActionCreationConfig.md#delete_key_cost)
 - [DeleteAccount](/RuntimeSpec/Actions.md#deleteaccountaction) uses
     - the base fee [`delete_account_cost`](/GenesisConfig/RuntimeFeeConfig/ActionCreationConfig.md#delete_account_cost)
-
+    - action receipt creation fee for creating Transfer to send remaining funds to `beneficiary_id`
+    - full transfer fee described in the corresponding item
+    
+## Gas tracking
+
+In `Runtime`, gas is tracked in the following fields of `ActionResult` struct:
+- `gas_burnt` - irreversible amount of gas spent on computations.
+- `gas_used` - includes burnt gas and gas attached to the new `ActionReceipt`s created during the method execution.
+- `gas_burnt_for_function_call` - stores gas burnt during function call execution. Later, owner of the corresponding contract gets 30% of it as a reward for a possibility to invoke the function.
```

Comment:
Owner is imprecise and potentially misleading. The contract account itself gets 30% of the gas burnt as a reward.